install.sh

```diff
@@ -164,6 +166,35 @@ if [ "$SENTRY_DATA_NEEDS_MIGRATION" ]; then
     "mkdir -p /tmp/files; mv /data/* /tmp/files/; mv /tmp/files /data/files; chown -R sentry:sentry /data"
 fi
 
+
+if [ ! -f "$RELAY_CREDENTIALS_JSON" ]; then
+    echo ""
+    echo "Generating Relay credentials..."
+
+    $dcr --user $(id -u) relay --config /etc/relay credentials generate --overwrite
+    chmod a+r $RELAY_CREDENTIALS_JSON
+    # display the contents of the relay directory (for debug purposes)
+    ls -al ./relay
```

Comment:
Let's remove these :)
```suggestion
```

Reply:
As mentioned before I think they don't hurt but I can remove them if you insist :-).

Reply:
They do hurt as they output unnecessary information to everyone, who are usually not interested in this information or be able to understand what this means.